code/__DEFINES/diseases.dm

```diff
@@ -12,8 +12,9 @@
 #define CAN_RESIST	(1<<2)
 
 // Spread Flags
-#define DISEASE_SPREAD_SPECIAL			(1<<0)
-#define DISEASE_SPREAD_NON_CONTAGIOUS	(1<<1)
+#define DISEASE_SPREAD_SPECIAL			(1<<-1)
```

Comment:
`(1<<-1)`

You uhhh, can't do this. All shifts must be positive.

Reply:
Hello sir?

Reply:
sorry, aint been on in awhile. I'll get to it in a bit.